### PR TITLE
Fix pay

### PR DESCRIPTION
**Pull request by gunsluo**

### CODE REVIEW

**Comment by gunsluo** [comment]
@clearcodecn  

Please review and merge.

**Comment by clearcodecn** [comment]
LGTM

File: pay.go [hunk 1]
```diff
@@ -84,11 +89,31 @@ const (
 
 // PayResponse is response when send a payment
 type PayResponse struct {
+	// The CodeUrl is returned when the merchant used Native
 	CodeUrl string `json:"code_url"`
+	// The CodeUrl is returned when the merchant used JSAPI APP
```
Comment (clearcodecn) [inline pay.go:94]: comment is incorrect.

File: pay.go [hunk 2]
```diff
@@ -84,11 +89,31 @@ const (
...
+	PrepayId string `json:"prepay_id"`
+	// The CodeUrl is returned when the merchant used H5
+	H5Url string `json:"h5_url"`
```
Comment (clearcodecn) [inline pay.go:97]: comment is incorrect.

File: pay.go [hunk 3]
```diff
@@ -84,11 +89,31 @@ const (
...
 }
 
 // Pay send a transaction and invoke wechat payment
```
Comment (clearcodecn) [inline pay.go:100]: comment should start with // Do

**Pull request merged**